publish.gradle

```diff
@@ -0,0 +1,71 @@
+apply plugin: 'maven-publish'
+
+ext.performRelease = !version.endsWith('SNAPSHOT')
+
+if (performRelease)
+    apply plugin: 'signing'
+
+task sourcesJar(type: Jar) {
+    from sourceSets.main.allJava
+    archiveClassifier = 'sources'
+}
+
+task javadocJar(type: Jar) {
+    from javadoc
+    archiveClassifier = 'javadoc'
+}
+
+publishing {
+    if (performRelease) {
+        repositories {
+            maven {
+                url "https://oss.sonatype.org/service/local/staging/deploy/maven2"
+                credentials {
+                    username ossrhUsername
+                    password ossrhPassword
+                }
+            }
+        }
```

Comment:
I'm not sure this is going to work. We cannot publish our username and password, It should be read from the settings.xml. Also we publish the artifact on our server and then it will take care to sync with Maven central.

In ORM they've just worked on the release scripts, we should check with @dreab8 and use a similar approach.

Reply:
@dreab8  could you link us to the part in the build of ORM that releases, please?

Reply:
By the way, we an skeep the configuration of the remote repositories for now and only keep the local one.
We will add them later as a separate issues

Reply:
@DavideD check  https://github.com/hibernate/hibernate-orm/blob/master/release/release.gradle and https://github.com/hibernate/hibernate-orm/blob/master/gradle/publishing-repos.gradle

Reply:
you may be also interested in https://github.com/hibernate/hibernate-orm/blob/master/gradle/publishing-pom.gradle and https://github.com/hibernate/hibernate-orm/blob/master/gradle/published-java-module.gradle

Reply:
THanks